server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -301,6 +303,9 @@ const orderMutations = {
       action: {
         type: new GraphQLNonNull(ProcessOrderAction),
       },
+      details: {
```

Comment:
I don't like `details` as name, I would propose `order` but it's already taken. Any other idea?

Reply:
Looking at the `editAccount` mutation, I think we should add `id` and `legacyId` to `OrderUpdateInput`

That way we can make the `order` args  an `OrderUpdateInput` instead of `OrderReferenceInput` and don't need a separated `details`.

Reply:
Yes, I think that makes sense. I'll update this mutation.